src/components/Dialog/Dialog.tsx

```diff
@@ -48,12 +48,19 @@ const Dialog: React.FC<DialogProps> = props => {
       onClose={dismissDialog}
       PaperComponent={Style.StyledPaper}
       fullScreen={fullScreen}
+      aria-labelledby="dialog-title"
+      disableScrollLock={false}
       {...otherProps}
     >
       <LockBodyScroll />
       <ButtonCloseBox onClickClose={closeDialog} />
```

Comment:
FWIW- The close button feels too close to the edges to me, and it's actually pushing the content of the dialog down in a way that doesn't seem desirable.

It seems better if I change the [ButtonContainer](https://github.com/covid-projections/covid-projections/blob/753cdf547a48f31749f095d29e36107f34d7e266/src/components/Dialog/Dialog.style.ts#L12) style to:
```
  position: absolute;
  top: 10px;
  right: 10px;
```

Before: 
![image](https://user-images.githubusercontent.com/206364/100569071-c90a5680-3281-11eb-866f-c3247e595855.png)

After:
![image](https://user-images.githubusercontent.com/206364/100568852-323d9a00-3281-11eb-9f45-54419a5c71f9.png)

Feel free to leave as-is if you disagree though. I guess we've been using this as-is in the recommends modal for a while.